### PR TITLE
Update corefxlab packages

### PR DESCRIPTION
**Pull request by pakrym**
- [ ] https://github.com/aspnet/KestrelHttpServer/pull/2124
- [ ] https://github.com/aspnet/SignalR/pull/998
- [ ] https://github.com/aspnet/IISIntegration/pull/454

### CODE REVIEW

**Comment by muratg** [comment]
cc @moozzyk

File: build/dependencies.props [hunk 1]
```diff
@@ -170,13 +175,13 @@
     <ExternalDependency Include="System.Management.Automation" Version="6.1.7601.17515" Source="$(DefaultNuGetFeed)"/>
     <ExternalDependency Include="System.Net.Http.WinHttpHandler" Version="4.4.0" Source="$(DefaultNuGetFeed)"/>
     <ExternalDependency Include="System.Net.Http" Version="4.3.2" Source="$(DefaultNuGetFeed)"/>
-    <ExternalDependency Include="System.Numerics.Vectors" Version="4.4.0" Source="$(DefaultNuGetFeed)"/>
+    <!--<ExternalDependency Include="System.Numerics.Vectors" Version="4.4.0" Source="$(DefaultNuGetFeed)"/>-->
```
Comment (natemcmaster) [inline build/dependencies.props]: Why is this commented out? If we don't need this, can you just remove it?

**Comment by pakrym** [comment]
This is work in progress, I'm still trying package versions and combinations. I'll ping this thread when ready for review.

**Comment by ryanbrandenburg** [comment]
Is there progress or news on this?

**Comment by pakrym** [comment]
Yep, working on it right now.

File: build/dependencies.props (hunk 1 above)
Comment (muratg) [reply build/dependencies.props]: Pinging on @natemcmaster's question.

File: build/dependencies.props [hunk 2]
```diff
@@ -43,6 +44,10 @@
     <ExternalDependency Include="Microsoft.NETCore.DotNetHostPolicy" Version="$(CoreSetupPackageVersion)" Source="$(DotNetCoreFeed)" Mirror="true" />
     <ExternalDependency Include="Microsoft.NETCore.DotNetHostResolver" Version="$(CoreSetupPackageVersion)" Source="$(DotNetCoreFeed)" Mirror="true" />
     <ExternalDependency Include="Microsoft.NETCore.Platforms" Version="2.1.0-preview1-25831-01" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Memory" Version="$(CoreFxVersion)" Source="$(DotNetCoreFeed)" Mirror="true" />
```
Comment (pranavkm) [inline build/dependencies.props]: Think these are alphabetically sorted. Also, there's another `System.Memory` in line 71

**Pull request merged**